espd-web/src/main/webapp/WEB-INF/weblogic.xml

```diff
@@ -26,13 +26,12 @@
 <weblogic-web-app xmlns="http://xmlns.oracle.com/weblogic/weblogic-web-app" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.oracle.com/weblogic/weblogic-web-app   http://xmlns.oracle.com/weblogic/weblogic-web-app/1.4/weblogic-web-app.xsd">
     <context-root>espd</context-root>
     <!--<session-descriptor>-->
-        <!--<cookie-path>/espd</cookie-path>-->
-        <!--<cookie-name>ESPD_APP</cookie-name>-->
-        <!--<cookie-http-only>true</cookie-http-only>-->
+    <!--<cookie-path>/espd</cookie-path>-->

```

Comment:
There's something wrong with spaces and tabs, which ones do you use in your editor?